docs/contributing/contributing-to-airbyte/developing-locally.md

```diff
@@ -29,8 +29,8 @@ This will build all the code and run all the unit tests.
 ## Run in `dev` mode with `docker-compose`
 
 ```bash
-./gradlew build
-docker-compose --env-file .env.dev -f docker-compose.yaml -f docker-compose.dev.yaml up
+./gradlew build composeBuild
+VERSION=dev docker-compose up
```

Comment:
This is incorrect.

The title says:
"Run in `dev` mode with `docker-compose`" not run with the dev images.

It sets all the options so you can develop locally and use the local file system

Reply:
the goal of that command is not to run it but to run it so you can develop